src/Bridge/Symfony/Translation/Extractor/EnumExtractor.php

```diff
@@ -113,7 +112,7 @@ public function extract($resource, MessageCatalogue $catalog)
 
                     $catalog->set($translationKey, $this->prefix . $translationKey, $this->domain);
                     $metadata = $catalog->getMetadata($translationKey, $this->domain) ?? [];
-                    $normalizedFilename = preg_replace('{[\\\\/]+}', '/', $file->getPathName());
+                    $normalizedFilename = preg_replace('{[\\\\/]+}', DIRECTORY_SEPARATOR, $file->getPathName());
```

Comment:
I think this one comes from https://github.com/symfony/symfony/blob/b285584e77e69ad9740711a12581c3fce46535ee/src/Symfony/Component/Translation/Extractor/PhpExtractor.php#L295 and should stay the same :)

Reply:
Reverting this change would make the testGenerateTranslations test fail because the path we assert on __DIR__.

We could revert this and use the preg_replace in the assert too.

``` 
$this->assertEquals(
            [
                'trans_readable_enum' => ['sources' => [ preg_replace('{[\\\\/]+}', '/',__DIR__ . '/Enum/ReadableEnum.php:readable_enum')]],
            ],
            $catalog->getMetadata('', 'messages_test')
        ); 
```

Reply:
Here is what's used in Symfony's tests: 
https://github.com/symfony/symfony/blob/b285584e77e69ad9740711a12581c3fce46535ee/src/Symfony/Component/Translation/Tests/Extractor/PhpExtractorTest.php#L116

Reply:
Adjusted